.github/workflows/code-quality.yml

```
@@ -0,0 +1,15 @@
+name: Code Quality Checks
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)